Cargo.toml

```diff
@@ -22,6 +22,8 @@ embedded-hal = "0.2.3"
 display-interface = "0.4"
 display-interface-i2c = "0.4"
 display-interface-spi = "0.4"
+typenum = "1.12.0"
```

Comment:
Small nit: `generic-array` reexports `typenum` so we don't need this additional dependency here.